src/Kunstmaan/NodeBundle/Helper/Menu/PageMenuAdaptor.php

```diff
@@ -183,20 +183,20 @@ private function getTreeNodes(
      */
     private function getActiveNodeIds($request)
     {
-        if (is_null($this->activeNodeIds)) {
+        if (null === $this->activeNodeIds) {
             if (stripos($request->attributes->get('_route'), 'KunstmaanNodeBundle_nodes_edit') === 0) {
                 $repo = $this->em->getRepository('KunstmaanNodeBundle:Node');
 
-                $currentNode         = $repo->findOneById($request->attributes->get('id'));
-                $parentNodes         = $repo->getAllParents($currentNode);
-                $this->activeNodeIds = array();
+                $currentNode = $repo->findOneById($request->attributes->get('id'));
+                $parentNodes = $repo->getAllParents($currentNode);
+                $this->activeNodeIds = [];
                 foreach ($parentNodes as $parentNode) {
                     $this->activeNodeIds[] = $parentNode->getId();
                 }
             }
         }
 
-        return (is_null($this->activeNodeIds) ? array() : $this->activeNodeIds);
+        return ($this->activeNodeIds ?? []);
```

Comment:
If this is reverted to PHP5 code, then we can use this PR in whatever version..

Reply:
Fixed ;)